http-server/src/main/java/io/airlift/http/server/HttpServer.java

```diff
@@ -244,7 +248,19 @@ public HttpServer(HttpServerInfo httpServerInfo,
             sslContextFactory.setIncludeCipherSuites(includedCipherSuites.toArray(new String[0]));
             sslContextFactory.setExcludeCipherSuites(excludedCipherSuites.toArray(new String[0]));
             sslContextFactory.setSecureRandomAlgorithm(config.getSecureRandomAlgorithm());
-            sslContextFactory.setWantClientAuth(true);
+            switch (clientCertificate) {
```

Comment:
Can you extract a method for this? Will remove the duplication and help with the factor in https://github.com/airlift/airlift/pull/853